src/huggingface_hub/hf_api.py

```diff
@@ -1352,10 +1398,14 @@ def repo_info(
                 An authentication token (See https://huggingface.co/settings/token)
             timeout (`float`, *optional*):
                 Whether to set a timeout for the request to the Hub.
+            files_metadata (`bool`, *optional*):
+                Whether or not to retrieve metadata for files in the repository
+                (size, LFS metadata, etc). Defaults to `False`.
 
         Returns:
-            `Union[SpaceInfo, DatasetInfo, ModelInfo]`: The repository
-            information.
+            `Union[SpaceInfo, DatasetInfo, ModelInfo]`: The repository information, as a
+            [`huggingface_hub.hf_api.DatasetInfo`], [`huggingface_hub.hf_api.ModelInfo`]
+            or [`huggingface_hub.hf_api.SpaceInfo`] object.
```

Comment:
not sure we need to expose it here

Reply:
Why not ?

Reply:
`repo_info` is just a helper exposing a common subset of params accross repo types.

But no strong opinion, we can keep it here if you think it's going to be useful